### DISCUSSION TITLE
vl2png dark theme not producing a dark background

### DISCUSSION THREAD

**Discussion by sanga** [vega/vega-lite#7447] [Q&A] [answered]
I am trying to use vega-lite (specifically `vl2png`) to plot data in my terminal however the background is unexpectedly white. Am I misunderstanding how this is supposed to work or is there a bug here?

![2021-05-08-090225_grim](https://user-images.githubusercontent.com/2088/117528841-edaaed00-afdc-11eb-9ce3-9804c11367ab.png)

This is against vl2png version 5.1.0 and I'm pretty sure the dark theme json is taken from the vega-themes repo. For reference the terminal is wez/wezterm on github but I don't believe it's relevant as Firefox also shows the background as white for that png

**Comment by domoritz** [comment, marked as answer]
That's a bug. Fixed in https://github.com/vega/vega-lite/pull/7449.

**Reply by sanga** [reply]
Thank you!
